### DISCUSSION TITLE
Possibility of having one csv datasource with the possiblilty of picking a csv to load in

### DISCUSSION THREAD

**Discussion by andrew-mcnaught** [grafana/grafana-csv-datasource#67] [Q&A] [answered]
Hello, 

I was wondering if there could be any work in the future to have a possibility of selecting csvs from a folder. In the current set up that I am using we have a lot of data sources currently, and it would be a bit frustrating to have to add a data source for every csv needed. Was wondering if in the future there could be any work on the possibility of having say a data source, this could be selected to a local folder (.e.g /path/to/csvfolder). Then when adding a panel and selecting this data source, it would give a drop down of the csvs within /path/to/csvfolder and from here you can select which csv you want to use? 

Look forward to hearing from you,
Thanks,
Andrew

**Comment by marcusolsson** [comment, marked as answer]
At this point, most of the features that are implemented are feature requests from community members. If this is important to you, then please create a feature request. The more people that are interested, the higher the priority.

Regarding the feature, I definitely see how that can be frustrating. We should figure out a good solution for this.

**Reply by andrew-mcnaught** [reply]
Thanks for response, this would be a very good use going forward I would say but uncertain how difficult that would be to implement. Created the feature request there, thanks again!
